hardhat.config.ts

```diff
@@ -121,6 +122,10 @@ const config: HardhatUserConfig = {
     tzketh: {
       url: `${QUICKNODE_ZKSYNC_SEPOLIA_API_KEY}`,
       accounts: [`${PRIVATE_KEY_FOR_V4_CONTRACT_DEPLOYMENT}`]
+    },
+    tbera: {
+      url: `${BARTIO_BERA_EXPLORER_API_KEY}`,
```

Comment:
Here you don't need Explorer API key. You need to add public rpc url for bera as this will be used to deploy the contracts

Reply:
fixed it. just realized when i was trying to run in my local